configs/envoy_double_proxy.template.yaml

```diff
@@ -0,0 +1,110 @@
+---
+{% macro listener(address,ssl,proxy_proto) -%}
+- address: "{{ address }}"
+  {% if ssl -%}
+  ssl_context:
+    alpn_protocols: h2,http/1.1
+    alt_alpn_protocols: http/1.1
+    cert_chain_file: certs/servercert.pem
+    private_key_file: certs/serverkey.pem
+  {% endif -%}
+  {% if proxy_proto -%}
+  use_proxy_proto: true
+  {% endif -%}
+  filters:
+  - name: http_connection_manager
+    config:
+      codec_type: auto
+      tracing:
+        operation_name: ingress
+      idle_timeout_s: 840
```

Comment:
duration fields dont specify the unit anymore so `idle_timeout_s` is now `idle_timeout` and the value should be a string with the unit appended so `840` should now be `840s`